project/project_repository.go

```diff
@@ -0,0 +1,184 @@
+package project
+
+import (
+	"log"
+
+	"golang.org/x/net/context"
+
+	"github.com/almighty/almighty-core/errors"
+	"github.com/almighty/almighty-core/gormsupport"
+	"github.com/jinzhu/gorm"
+	satoriuuid "github.com/satori/go.uuid"
+)
+
+// NewProjectRepository creates a new project repo
+func NewProjectRepository(db *gorm.DB) *GormProjectRepository {
```

Comment:
Rename to `NewRepository` and rename the file `project/project_repository.go` to `project/repository.go` as it  is mentioning projects one time too often ;)